CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+7.2.12
+-----
+* Bump required omniauth-shopify-oauth2 version to 1.2.0.
```

Comment:
Can you please add the change from https://github.com/Shopify/shopify_app/pull/430 ?
Maybe `* Always expect params[:shop] to be a string`.